test/Microsoft.Diagnostics.EventFlow.Outputs.Tests/ElasticSearchOutputTests.cs

```diff
@@ -43,7 +43,13 @@ public void VerifyAllConfigOptionsMappedToConfigObject()
                                     ["type"] = "date_nanos"
                                 }
                             }
-                        }
+                        },
+                        ["proxy"] = new Dictionary<string, object>
```

Comment:
It is great that we could cover this case here. I am just wondering, is it common for a proxy to be provided without username and password? What do you think to add some unit tests to cover that when userName or password being null or empty?

Reply:
For our on-prem environment there's no username/password for the proxy. And the proxy is plain http. The proxy is being used to limit which internet sites can be accessed from our production servers.

Reply:
Like @srothkin said, empty/null username and passwords are valid use cases.

Reply:
@karolz-ms Your call. But what I meant was to have unit test cover when userName/password is null and the code keeps working. In other words, probably no exception when userName or password is empty.